composer.json

```diff
@@ -240,7 +240,8 @@
                 "scheduled_transitions module integration issue - https://www.drupal.org/project/permissions_by_term/issues/3065986#comment-13171384": "https://www.drupal.org/files/issues/2019-07-05/permissions_by_term-issue-3065986-D8.6.16.patch"
             },
             "drupal/password_policy": {
-                "Add password validation to the user entity as a constraint - https://www.drupal.org/project/password_policy/issues/2941526#comment-13525843": "https://www.drupal.org/files/issues/2020-03-27/password_policy-move-password-validation-to-user-entity-2941526-8-D8.patch"
+                "Add password validation to the user entity as a constraint - https://www.drupal.org/project/password_policy/issues/2941526#comment-13525843": "https://www.drupal.org/files/issues/2020-03-27/password_policy-move-password-validation-to-user-entity-2941526-8-D8.patch",
+                "Email user when password expires - https://www.drupal.org/project/password_policy/issues/2997543#comment-14157182": "https://www.drupal.org/files/issues/2021-07-07/2997543-31.patch"
```

Comment:
This patch has been committed as of 11/9/2021. This will need to come out after the next stable release.

Reply:
Issue has been closed and marked as fixed. 
https://www.drupal.org/project/password_policy/issues/2997543